pkg/deploy/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -63,6 +63,8 @@ type DeploymentConfigController struct {
 
 	// queue contains deployment configs that need to be synced.
 	queue workqueue.RateLimitingInterface
+	// instantiateQueue contains deployment config that should be instantiated.
+	instantiateQueue workqueue.RateLimitingInterface
```

Comment:
@kargakis i didn't want to pollute the main DC controller queue with instantiate calls, that's why I made separate queue just for instantiating (terrible word).